server/controllers/userController.js

```diff
@@ -14,7 +14,7 @@ dotenv.config();
 
 class UserController {
   static async signUp(req, res) {
-    const { firstName, lastName, email, password } = req.body;
+    const { firstName, lastName, email, password, username } = req.body;
```

Comment:
When was username added?

Reply:
And why was it added with no intention of being unique?

Reply:
That's an oversight, will either remove it, or make it unique